### DISCUSSION TITLE
FleetDM 5.3.0 software inventory does not work

### DISCUSSION THREAD

**Discussion by hackecu** [Security-Onion-Solutions/securityonion#8647] [General] [answered]
I have installed the osquery package and I can perform queries correctly, including software inventory queries. 
![image](https://user-images.githubusercontent.com/112084710/187660997-43b14bfb-020c-484c-b339-517a8df21ee9.png)

But in the software section I get the error message "Software inventory is disabled".
![image](https://user-images.githubusercontent.com/112084710/187661202-b7434183-1390-44aa-8c72-c0efb5e95e4e.png)
I have followed the instructions to enable it and the error persists.
![image](https://user-images.githubusercontent.com/112084710/187661393-cdfacf79-8e9b-440f-9666-5f96da17ab92.png)
![image](https://user-images.githubusercontent.com/112084710/187661492-ae2e5b9d-efa7-4616-9fda-b00cc5de3266.png)

**Comment by defensivedepth** [comment, marked as answer]
Some context:

We initially disabled it to allow time for it to mature a bit: https://github.com/Security-Onion-Solutions/securityonion/discussions/7643#discussioncomment-2440468

Here is how you can manually enable it: https://github.com/Security-Onion-Solutions/securityonion/discussions/7844#discussioncomment-2647835
